lib/text/listener.js

```diff
@@ -272,6 +348,24 @@ class Importer extends SHRParserListener {
     return concept;
   }
 
+    processInlineValue(ctx) {
+        var cs;
+        const csAlias = ctx.CODE().getText();
```

Comment:
This isn't right because the `CODE` token is always just the code part (_without_ the system) -- for example, `#food`.  I think you understood this below (in line 361), but maybe missed it here?

This is the case where you basically have to make up a code system.  I'd just base it on the namespace -- so if the namespace is `shr.foo`, and the value set the code is defined in is `Bar`, then your code system could be `http://standardhealthrecord.org/cs/shr/foo/Bar`.